hooks/pre/30-upgrade-postgresql.rb

```diff
@@ -0,0 +1,18 @@
+def postgresql_10_upgrade
+  execute('foreman-maintain service start --only=postgresql')
```

Comment:
Is `f-m` required on a plain Foreman machine these days? I don't think it is.

Reply:
I will need to update the packaging to require it by the foreman-installer. Not so much because of this line, but I need to be able to stop all services for all scenarios.

Reply:
foreman-maintain isn't packaged for Debian so the installer can't depend on it.

Reply:
This hook is technically limited to EL7 only, so the EL7 installer packaging can depend on it.